legal-api/src/legal_api/services/authz.py

```diff
@@ -135,30 +135,30 @@ def get_allowable_filings_dict():
         'staff': {
             Business.State.ACTIVE: {
                 'adminFreeze': {
-                    'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC'],
+                    'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC', 'C', 'CBEN', 'CUL', 'CCC'],
                 },
                 'agmExtension': {
-                    'legalTypes': ['BC', 'BEN', 'ULC', 'CC'],
+                    'legalTypes': ['BC', 'BEN', 'ULC', 'CC', 'C', 'CBEN', 'CUL', 'CCC'],
                     'blockerChecks': {
                         'business': [BusinessBlocker.DEFAULT, BusinessBlocker.NOT_IN_GOOD_STANDING]
                     }
                 },
                 'agmLocationChange': {
-                    'legalTypes': ['BC', 'BEN', 'ULC', 'CC'],
+                    'legalTypes': ['BC', 'BEN', 'ULC', 'CC', 'C', 'CBEN', 'CUL', 'CCC'],
                     'blockerChecks': {
                         'business': [BusinessBlocker.DEFAULT, BusinessBlocker.NOT_IN_GOOD_STANDING]
                     }
                 },
                 'alteration': {
-                    'legalTypes': ['BC', 'BEN', 'ULC', 'CC'],
+                    'legalTypes': ['BC', 'BEN', 'ULC', 'CC', 'C', 'CBEN', 'CUL', 'CCC'],
                     'blockerChecks': {
                         'business': [BusinessBlocker.DEFAULT, BusinessBlocker.IN_DISSOLUTION]
                     }
                 },
                 'amalgamationApplication': {
                     'businessRequirement': BusinessRequirement.NO_RESTRICTION,
                     'regular': {
-                        'legalTypes': ['BEN', 'BC', 'ULC', 'CC'],
+                        'legalTypes': ['BEN', 'BC', 'ULC', 'CC', 'C', 'CBEN', 'CUL', 'CCC'],
```

Comment:
This may not be valid. amalgamationApplication filing is used to create a new business. Through amalgamationApplication a user can't create continuation in business. Please confirm with Naveen.

Reply:
Will check with Naveen

Reply:
Checked with Naveen. He confirmed, user can create an amalgamationApplication from continued in business

Reply:
yeah! this is to create a new business which will end up in these legal types (from will be inside amalgamationApplication['amalgamatingBusinesses'])